webapp-django/crashstats/documentation/templates/documentation/supersearch/home.html

```diff
@@ -0,0 +1,389 @@
+                        <td><code>__true__</code></td>
+                        <td>boolean</td>
+                        <td>The field has a value that is truthy.</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <h2>Data types</h2>
+
+            <p>Each filter has a data type, that corresponds to the type of data the associated field can receive. Those types have a direct impact on the operators you can use for each field, as shown above. Here are the existing data types:</p>
+
+            <table>
+                <thead>
+                    <tr>
+                        <th>Data type</th>
+                        <th>Operators</th>
+                        <th>Description</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>enum</td>
+                        <td><code></code></td>
+                        <td>Simple type for strings, relies on inputs being turned into "terms". </td>
+                    </tr>
+                    <tr>
+                        <td>string</td>
+                        <td><code></code>, <code>~</code>, <code>=</code>, <code>$</code>, <code>^</code>, <code>__null__</code></td>
+                        <td>Advanced type for strings, gives a lot more options. </td>
+                    </tr>
+                    <tr>
+                        <td>number</td>
+                        <td><code></code>, <code>&gt;</code>, <code>&gt;=</code>, <code>&lt;</code>, <code>&lt;=</code></td>
+                        <td>Type for all numbers, underlying data can be float, int, long&hellip; </td>
+                    </tr>
+                    <tr>
+                        <td>date</td>
+                        <td><code>&gt;</code>, <code>&gt;=</code>, <code>&lt;</code>, <code>&lt;=</code></td>
+                        <td>Type for all dates, underlying data can be date or datetime. </td>
+                    </tr>
+                    <tr>
+                        <td>boolean</td>
+                        <td><code>__true__</code></td>
+                        <td>Type for fields that can be true or false.</td>
+                    </tr>
+                    <tr>
+                        <td>flag</td>
+                        <td><code>__null__</code></td>
+                        <td>Type for fields that either have a value (generally <code>1</code>) or are missing.</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <h1 id="controls">Controls</h1>
+
+            <p>Along with filters, Super Search exposes a set of "meta" parameters. Their names start with an underscore, like <code>_results_number</code>. These parameters control various aspects of the results in the <code>hits</code> key. They are pretty straight forward and are well described in the <a href="{{ url('documentation:supersearch_api') }}">API page</a>.</p>
+
```

Comment:
This is too brief. I'd like to see a list of them. Listing them here would make it easy to mentally separate that out from the more advanced aggregation keys.